src/frontend/src/pages/ProjectDetailPage/ProjectForm/LinksEditView.tsx

```diff
@@ -17,6 +18,15 @@ const LinksEditView: React.FC<{
   remove: UseFieldArrayRemove;
 }> = ({ ls, register, append, remove, watch }) => {
   const { isLoading, isError, error, data: linkTypes } = useAllLinkTypes();
+
+  useEffect(() => {
+    requiredLinkTypeNames.forEach((linkTypeName) => {
+      if (links.some((link) => link.linkTypeName === linkTypeName)) return;
+      append({ linkId: '-1', url: '', linkTypeName });
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [linkTypes]);
```

Comment:
not sure how this works rn, since it's calling requiredLinkTypeNames and links before they're declared, I would try to reshuffle things so you don't have to eslint disable this line